support-frontend/assets/pages/subscriptions-landing/copy/subscriptionCopy.tsx

```diff
@@ -52,6 +59,32 @@ const getDisplayPrice = (
 	return `${currency}${fixDecimals(price)}/${billingPeriod}`;
 };
 
+const getDigitalPlusDisplayPrice = (
+	countryGroupId: CountryGroupId,
+	billingPeriod: BillingPeriod,
+): string => {
+	const currencyKey = detect(countryGroupId);
+	const product = productCatalog['DigitalSubscription'];
+	const price = product?.ratePlans[billingPeriod]?.pricing[currencyKey];
+	if (!price) {
+		return '';
```

Comment:
If we ever got to this point, would the button be rendered? (And if it is, should we prevent it?)